common/app/model/content.scala

```diff
@@ -26,14 +26,9 @@ import scala.util.Try
 /**
  * a combination of CAPI content and things from facia tool, in one place
  */
-class Content protected (val apiContent: ApiContentWithMeta) extends Trail with MetaData with ShareLinks {
+class Content protected (delegate: com.gu.contentapi.client.model.Content) extends Trail with MetaData with ShareLinks {
 
-  lazy val delegate: ApiContent = apiContent.delegate
-
-  lazy val snapType: Option[String] = apiContent.metaData.flatMap(_.snapType).filter(_.nonEmpty)
-  lazy val snapCss: Option[String] = apiContent.metaData.flatMap(_.snapCss).filter(_.nonEmpty)
-  lazy val snapUri: Option[String]  = apiContent.metaData.flatMap(_.snapUri).filter(_.nonEmpty)
-  lazy val snapHref: Option[String] = apiContent.metaData.flatMap(_.href).filter(_.nonEmpty)
+  val apiContent: com.gu.contentapi.client.model.Content = delegate
```

Comment:
Yes, I was having problems merging the two. I'm not sure why, might try to fix it again now